spec/support/shared_contexts/cli/when_main_cli_handler_setup_context.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+shared_context 'when main cli handler setup' do
+  let(:main_handler) do
+    inst = Codeowners::Cli::Main.new
+
+    inst.options = options
+    inst.instance_variable_set(:@config, config)
+    inst.instance_variable_set(:@checker, checker)
+    inst.instance_variable_set(:@owners_list_handler, owners_list_handler)
+
+    inst
+  end
+  let(:options) { {} }
+  let(:config) do
+    conf = Codeowners::Cli::Config.new
+    conf.owner(default_owner)
+
+    conf.config
+  end
+  let(:default_owner) { '@toptal/bootcamp' }
+  let(:new_owner) { '@toptal/ninjas' }
+  let(:folder_name) { '.' }
+  let(:from) { 'HEAD' }
+  let(:to) { 'HEAD' }
```

Comment:
This `from` and `to` looks like defaults (in fact we missed for `to` in the Checker initializer` and they're not being overridden or tested. I'd simply remove them and fix the code since we don't refer them here.